ExpressWash/ExpressWash/ReceiptDetailViewController.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  ReceiptDetailViewController.swift
+//  ExpressWash
+//
+//  Created by Bobby Keffury on 6/3/20.
+//  Copyright © 2020 Bobby Keffury. All rights reserved.
+//
+
+import UIKit
+
+class ReceiptDetailViewController: UIViewController {
+
+    // MARK: - Properties
+
+    let washerController = WasherController()
+    var job: Job?
+    var rating: Int?
+    var star = UIImage(systemName: "star")
+    var starFill = UIImage(systemName: "star.fill")
+
+    // MARK: - Outlets
+
+    @IBOutlet weak var washDateLabel: UILabel!
+    @IBOutlet weak var addressLabel: UILabel!
+    @IBOutlet weak var cityStateLabel: UILabel!
+    @IBOutlet weak var timeTakenLabel: UILabel!
+    @IBOutlet weak var beforeImageView: UIImageView!
+    @IBOutlet weak var afterImageView: UIImageView!
+    @IBOutlet weak var washerProfileImageView: UIImageView!
+    @IBOutlet weak var washerNameLabel: UILabel!
+    @IBOutlet weak var washerRatingLabel: UILabel!
+    @IBOutlet weak var washerAboutMeTextView: UITextView!
+
+    @IBOutlet weak var oneStar: UIButton!
+    @IBOutlet weak var twoStar: UIButton!
+    @IBOutlet weak var threeStar: UIButton!
+    @IBOutlet weak var fourStar: UIButton!
+    @IBOutlet weak var fiveStar: UIButton!
+
+    @IBOutlet weak var rateWasherButton: UIButton!
+
+    // MARK: - Views
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        updateViews()
+    }
+
+    // MARK: - Methods
+
+    private func updateViews() {
+        guard let job = job else { return }
+        guard let user = job.washer?.user else { return }
+
+        addressLabel.text = job.address
+        cityStateLabel.text = "\(job.city.capitalized), \(job.state.capitalized)"
+        // Time Taken
+        // Images
+        washerNameLabel.text = "\(user.firstName.capitalized) \(user.lastName.capitalized)"
+        washerRatingLabel.text = "★ \(job.washer!.washerRating)"
+        washerAboutMeTextView.text = "About your washer:/n/n\(job.washer!.aboutMe ?? "")"
+    }
+
+    private func oneStarFill() {
```

Comment:
probably could use a switch instead of five different functions, but no big deal